src/main/java/org/jenkins/tools/test/exception/WrappedPluginCompatibilityException.java

```diff
@@ -0,0 +1,17 @@
+package org.jenkins.tools.test.exception;
+
+/** A {@code RuntimeException} that wraps a PluginCompatibilityTesterException. */
+public class WrappedPluginCompatibilityException extends RuntimeException {
```

Comment:
[nit] This is like the JDK's `UncheckedIOException`, so I think it would be nice to use a similar name here e.g. `UncheckedPluginCompatibilityTesterException`.

Reply:
I got rid of this class entirely in one of my commits.